rewriter/DefDelegator.cc

```diff
@@ -0,0 +1,74 @@
+#include "rewriter/DefDelegator.h"
+#include "ast/Helpers.h"
+#include "core/GlobalState.h"
+#include "rewriter/Util.h"
+#include <optional>
+
+using namespace std;
+
+namespace sorbet::rewriter {
+
+vector<ast::TreePtr> DefDelegator::run(core::MutableContext ctx, const ast::Send *send) {
+    vector<ast::TreePtr> empty;
```

Comment:
Could you rename this vector to `methodStubs`, and remove the declaration of `methodStubs` on line 52?